js/apps/admin-ui/src/clients/authorization/AuthorizationExport.tsx

```diff
@@ -68,25 +65,14 @@ export const AuthorizationExport = () => {
         role="manage-authorization"
         className="pf-u-mt-lg"
       >
-        <FormGroup
+        <TextAreaControl
+          name="authDetails"
           label={t("authDetails")}
-          labelIcon={
-            <HelpItem
-              helpText={t("authDetailsHelp")}
-              fieldLabelId="authDetails"
-            />
-          }
-          fieldId="client"
-        >
-          <KeycloakTextArea
-            id="authorizationDetails"
-            readOnly
-            resizeOrientation="vertical"
```

Comment:
don't we need to still have this `resizeOrientation`?

Reply:
Thanks @edewit I'll add it in my next PR.